api/v1/configurationpolicy_types.go

```diff
@@ -187,8 +187,21 @@ type ObjectTemplate struct {
 	// ObjectDefinition defines required fields for the object
 	// +kubebuilder:pruning:PreserveUnknownFields
 	ObjectDefinition runtime.RawExtension `json:"objectDefinition"`
+
+	// RecordDiff specifies whether (and where) to log the diff between the object on the
+	// cluster and the objectDefinition in the policy. Defaults to "None".
+	// +kubebuilder:default:=None
+	RecordDiff RecordDiff `json:"recordDiff,omitempty"`
```

Comment:
Does `omitempty` apply here? I think it'd have to be a pointer for that to work since the default is set to `None`.

Reply:
I think we probably want to remove `// +kubebuilder:default:=None` and treat an empty string or this set to `None` as equivalent in the code.